cohorts/variant_stats.py

```diff
@@ -158,6 +160,30 @@ def maf_somatic_variant_stats(variant, variant_metadata):
         normal_stats = _maf_variant_stats(variant, variant_metadata, prefix="n")
     return SomaticVariantStats(tumor_stats=tumor_stats, normal_stats=normal_stats)
 
+def _vcf_is_strelka(variant_file):
+    """Return True if variant_file given is in strelka format
+    """
+    return "strelka" in variant_file.lower()
+
+def _vcf_is_maf(variant_file):
+    """Retrun True if variant_file given is in .maf format
+    """
+    return ".maf" in variant_file.lower()
+
+def _vcf_is_mutect(variant_file):
+    """Return True if variant_file give is in mutect format
+    """
+    if "mutect" in variant_file.lower():
+        return True
+    else:
```

Comment:
This is a good point. The error i'm trying to catch here is one caused by the meta-data being a different structure depending on the file type. I am about to push an edit that will try to isolate it better. 

Curious if you know of a better way to handle this ? ie Confirming that the item is a list of dicts, and that this dict contains a certain key seems like overkill.